site/components/Home.tsx

```diff
@@ -1,46 +1,46 @@
 'use client'
 
-import { Button, Link, HStack, Box, Stack, Heading } from "@chakra-ui/react"
+import { Button, Link, HStack, Box, Stack, Heading } from '@chakra-ui/react'
 import { useTheme } from 'nextra-theme-docs'
 import Prover from './Prover'
 import Why from './Why'
 import Navbar from './Navbar'
-import { ColorModeSync } from "./ColorModeSync"
+import { ColorModeSync } from './ColorModeSync'
 
 const Home = ({ children }) => {
-    return (
-        <Box maxW='container.lg' margin='0 auto'>
-            <ColorModeSync />
-            <Stack spacing='6em'>
-                <Box py={3}>
-                    <Navbar />
-                </Box>
-                <Stack spacing='10em'>
-                    <Stack spacing={6}>
-                        <Heading as='h1' fontSize={['3em', '4em']}>
+  return (
+    <Box maxW='container.lg' margin='0 auto'>
+      <ColorModeSync />
+      <Stack spacing='6em'>
+        <Box py={3}>
+          <Navbar />
+        </Box>
+        <Stack spacing='10em'>
+          <Stack spacing={6}>
+            <Heading as='h1' fontSize={['3em', '4em']}>
                             TypeScript for<br /> Zero Knowledge
```

Comment:
And this line

Reply:
I'll update the changes from `main` and double-check on this.